tensorrt_llm/_torch/attention_backend/inkling_triton.py

```
@@ -0,0 +1,520 @@
+
+    Returns ``[total_tokens, num_heads, head_dim]`` in q's dtype.
+    """
+    # The kernels index the head_dim with an implicit stride-1 last axis, so the
+    # inputs must be contiguous. ``v`` in particular reaches here non-contiguous:
+    # it is the fused-qkv v slice run through the short conv, and (unlike ``k``)
+    # never passes through ``apply_qk_norm``'s reshape, so it keeps the qkv row
+    # stride. ``.contiguous()`` is a no-op for already-contiguous q/k.
+    q = q.contiguous()
+    k = k.contiguous()
+    v = v.contiguous()
+    total_tokens, num_heads, head_dim = q.shape
```

Ruff (0.16.0)

[warning] Unpacked variable `total_tokens` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)